packages/nextjs/app/builders/0x8eE31084d2914fA84Baae3460093564934837898/page.tsx

```diff
@@ -0,0 +1,94 @@
+            </div>
+          </div>
+        </div>
+
+        {/* content column */}
+        <div className="w-1/2 bg-gradient-to-r from-secondary to-base-100 dark:from-secondary dark:to-base-300 rounded-r-3xl flex flex-col justify-center p-8">
+          <div className="flex justify-between items-center">
+            <h1 className="text-5xl font-semibold">
+              <span className="bg-clip-text text-black dark:text-white bg-gradient-to-r from-base-400 to-base-100 via-base-150 dark:from-secondary dark:via-secondary dark:to-secondary">
+                PenaMaster
+              </span>
+              <span className="space-y-1">
+                <Address address="0x8eE31084d2914fA84Baae3460093564934837898" />
+              </span>
+            </h1>
+            <div className="flex gap-3">
+              <a href="https://github.com/penaMaster" target="_blank" rel="noopener noreferrer">
+                {/* eslint-disable-next-line @next/next/no-img-element */}
+                <img
+                  src="https://github.githubassets.com/images/modules/logos_page/GitHub-Mark.png"
+                  alt="GitHub"
+                  className="h-8 w-8"
+                />
+              </a>
+              <a
+                href="https://app.buidlguidl.com/builders/0x8eE31084d2914fA84Baae3460093564934837898"
+                target="_blank"
+                rel="noopener noreferrer"
+              >
+                <BuidlGuidlLogo className="h-8 w-8" />
+              </a>
+              <a href="https://t.me/penamasterr" target="_blank" rel="noopener noreferrer">
+                {/* eslint-disable-next-line @next/next/no-img-element */}
+                <img
+                  src="https://upload.wikimedia.org/wikipedia/commons/8/82/Telegram_logo.svg"
+                  alt="Telegram"
+                  className="h-8 w-8"
+                />
+              </a>
+              
+            </div>
+          </div>
+          <div className="mt-4">
+            <p className="flex items-start gap-3 text-gray-900 dark:text-gray-200">
+              <RocketLaunchIcon className="h-8 w-8 text-blue-500 dark:text-blue-400 hover:scale-125 transition-transform duration-300" />
+              <span>Web3 Enthusiast</span>
+            </p>
+            <p className="flex items-start gap-3 text-gray-900 dark:text-gray-200">
+              <CodeBracketIcon className="h-8 w-8 text-green-500 dark:text-green-400 hover:scale-125 transition-transform duration-300" />
+              <span>Full-stack + Smart Contract Developer</span>
+            </p>
+            
+            <p className="flex items-start gap-3 text-gray-900 dark:text-gray-200">
+              <AcademicCapIcon className="h-8 w-8 text-purple-500 dark:text-purple-400 hover:scale-125 transition-transform duration-300" />
+              <span>Learning Solidity</span>
+            </p>
+          </div>
+        </div>
+      </div>
+    </>
+  );
+};
+
+export default page;
```

Comment:
Prettier adds a new line after the end of the file. You usually want to configure your editor so that whenever you save prettier formatting is called (so it auto-formats de file)